src/js/edu-benefits/1990e/config/form.js

```diff
@@ -68,9 +68,7 @@ const formConfig = {
           path: 'benefits/eligibility',
           title: 'Benefits eligibility',
           uiSchema: {
-            'view:benefitDescription': {
-              'ui:description': eligibilityDescription
-            },
+            'ui:description': eligibilityDescription,
```

Comment:
Good to know we don't have to wrap this in a `view:` field. Are `view:` fields only needed for actual form fields, or when other fields are nested within them?